src/custom-types.json

```diff
@@ -501,6 +501,12 @@
   "application/zip": {
     "compressible": false
   },
+  "application/dicom": {
+    "source": [
+        "https://en.wikipedia.org/wiki/DICOM"
```

Comment:
This needs to be a link to a primary source; Wikipedia is even specifically called out as not acceptable in the README: https://github.com/jshttp/mime-db#direct-inclusion

Reply:
See instead:
* https://www.dicomstandard.org/concepts
  > DICOM files typically have a .dcm file extension if they are not part of a DICOM media (which requires them to be without extension). 
* [DICOM PS3.12 2023c - Media Formats and Physical Media for Media Interchange](https://www.dicomstandard.org/standards/view/media-formats-and-physical-media-for-media-interchange)
  > * "name" is constructed from the last DICOM File ID component (that means the "file name" without "path" information) and the extension ".dcm" (except for the DICOMDIR).
  > 
  > For example: "I00023.dcm"
* [RFC 3240](https://www.rfc-editor.org/info/rfc3240)
  > 2. File extension(s): ".dcm" is recommended for files saved to disk (other than DICOMDIR)

Reply:
Additionally, `"application/dicom"` is already present in [`src/iana-types.json`](https://github.com/jshttp/mime-db/blob/master/src/iana-types.json):
https://github.com/jshttp/mime-db/blob/a76e5a824c228e2e58363c9404e42a54ee1d142f/src/iana-types.json#L607-L612

You should probably add `"extensions": ["dcm"]` at the above location, like #295.